script/default-site

```diff
@@ -12,7 +12,7 @@ workdir=$(pwd)
 echo "$0: creating new default site"
 if [[ "$(uname -s)" != "Linux" ]]
 then 
-  exec exe/jekyll new tmp/default-site
+   `exe/jekyll new tmp/default-site`
```

Comment:
Ah.. I was just testing this change... 
exec did not seem to run the code that followed.. 
I'd reverted back locally.. just didnt push to remote..